.github/workflows/ci.yml

```diff
@@ -529,6 +529,12 @@ jobs:
         env:
           DD_API_KEY: ${{ secrets.DD_API_KEY }}
 
+      - name: Run APPSEC_IP_BLOCKING scenario
+        if: ${{ matrix.library != 'java' || matrix.weblog-variant == 'spring-boot' }}
```

Comment:
why not running it on other weblogs, just skipping the failing test ?

Reply:
There is currently a manual flag required on the java cli to enable remote config. I only enabled it on spring-boot. As the next version of the tracer will remove this flag, we can wait the release and update this PR or open another PR later. WDYT?

Reply:
Given the current status in Java, I think it would be more convenient for us to:
* Change this PR to add CLI to enable remote config to all relevant weblog variants, and remove this condition here.
* We'll open a separate PR to remove the enable remote config CLI parameter from all Java weblog variants soon.

Reply:
I would also prefer to add `@missing_feature(context.library == "java" and context.weblog_variant != "spring-boot")` on the corresponding test, it'll be easier for me to handle (I'll need to add this scenario in nightly runs)

Edit : see below he good line for this.